code/modules/mob/living/simple_mob/subtypes/vore/corrupt_hounds.dm

```diff
@@ -71,13 +71,13 @@
 
 	say_list_type = /datum/say_list/corrupthound_prettyboi
 
-/mob/living/simple_mob/vore/corrupthound/isSynthetic()
+/mob/living/simple_mob/vore/aggresive/corrupthound/isSynthetic()
 	return TRUE
 
-/mob/living/simple_mob/vore/corrupthound/speech_bubble_appearance()
+/mob/living/simple_mob/vore/aggresive/corrupthound/speech_bubble_appearance()
```

Comment:
Can actually change all this by making the mob a simple_mob/mechanical/corrupthound

Reply:
intent is to keep all voremobs under /vore/ subtype